src/test/scala/org/ua/ilm/datamining/crawlers/youtube/parse/ChannelParserTest.scala

```diff
@@ -1,15 +1,17 @@
-    val jsonPars = YoutubeChannelParser
+    val jsonPars = new ChannelParser
```

Comment:
YoutubeChannelParser is good naming. Actyally your in-item is in yt json format. I think there is no need in removing this prefix